tests/backends/oracle/test_introspection.py

```diff
@@ -33,3 +33,13 @@ def test_get_sequences_manually_created_index(self):
                 # Recreate model, because adding identity is impossible.
                 editor.delete_model(Square)
                 editor.create_model(Square)
+
+    def test_get_table_description(self):
+        with connection.cursor() as cursor:
+            cursor.execute("SELECT * FROM %s" % Square._meta.db_table)
+            description = connection.introspection.get_table_description(
+                cursor, Square._meta.db_table
+            )
+            self.assertEqual(len(description), 2)
+            self.assertEqual(description[0][0], "ID")
+            self.assertEqual(description[1][0], "NAME")
```

Comment:
This is not a proper regression test.

Reply:
Regression tests that I should apply should be carried out in the tests.py file in the tests/admin_changelist path?

Reply:
Why in the `admin_changelist`? :thinking:  This bug has nothing to do with the admin.

Reply:
I wanted to make sure I was understanding the change where it should be corrected, since in the vscode search engine I looked for regression tests and most of them appeared in that section, thank you very much for guiding me, have a good day